core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -123,6 +123,7 @@ registry.ci.openshift.org/origin/4.10:gcp-pd-csi-driver-operator quay.io/openshi
 registry.ci.openshift.org/origin/4.10:gcp-pd-csi-driver-operator-test quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:4.10 quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:4.10.0 quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.10:ghostunnel quay.io/openshift/origin-ghostunnel:4.10 quay.io/openshift/origin-ghostunnel:4.10.0 quay.io/openshift/origin-ghostunnel:latest
 registry.ci.openshift.org/origin/4.10:grafana quay.io/openshift/origin-grafana:4.10 quay.io/openshift/origin-grafana:4.10.0 quay.io/openshift/origin-grafana:latest
+registry.ci.openshift.org/origin/4.10:graphdata quay.io/openshift/origin-graphdata:4.10 quay.io/openshift/origin-graphdata:4.10.0 quay.io/openshift/origin-graphdata:latest
```

Comment:
We already have [one graph-data container being mirrored in this file][1]:

```
$ git --no-pager grep cincinnati-graph-data core-services/image-mirroring/openshift/mapping_origin_4_10
core-services/image-mirroring/openshift/mapping_origin_4_10:registry.ci.openshift.org/origin/4.10:cincinnati-graph-data-container quay.io/openshift/origin-cincinnati-graph-data-container:4.10 quay.io/openshift/origin-cincinnati-graph-data-container:4.10.0 quay.io/openshift/origin-cincinnati-graph-data-container:latest
```

Can you explain how this new image differs from that one?

[1]: https://github.com/openshift/release/blob/5fdc1a0a8fde9a67a2bf836a9431611f1ad5e892/core-services/image-mirroring/openshift/mapping_origin_4_10#L35

Reply:
`cincinnati-graph-data-container` is a similar image built from https://github.com/openshift/cincinnati-operator/blob/master/dev/Dockerfile. This image is used in cincinnati-operator testing, however its being built only when an operator commit lands - so its not being updated frequently enough.

If this PR lands I'll update operator code to replace cincinnati-graph-data-container with `graphdata` (and rename these properly)

Reply:
Graphdata is too generic to be used as the name of the image.  What is the full purpose of this image within the origin release (it's not just to have data about graphs).  It's not really that interesting to add `-container`.  Is this test specific?  Is this an authoritative source of info?  Non-authoritative (trailing)?  I need to clearly understand the goal to be able to approve a name.

Reply:
Hmm, we probably don't need this image in origin release.
Reworked this to promote to `cincinnati-ci-public` namespace instead